vespaclient-core/src/main/java/com/yahoo/feedhandler/VespaFeedHandler.java

```diff
@@ -100,6 +165,9 @@ public HttpResponse handle(HttpRequest request, RouteMetricSet.ProgressCallback
             }
 
             sender.done();
+            if (feedAccess instanceof ThreadedFeedAccess) {
+                ((ThreadedFeedAccess)feedAccess).close();
+            }
```

Comment:
Perhaps the SimpleFeedAccess interface could have a close method instead?

Reply:
And if you are making changes I suggest making ThreadedFeedAccess a top-level class.

Reply:
What does top-level class imply ?

Reply:
Not nested inside another class.